### ISSUE TITLE
Duplicate events when resizing

### ISSUE THREAD

**Issue by nomrik** [intljusticemission/react-big-calendar#1441]
#### Do you want to request a _feature_ or report a _bug_?
Bug

#### What's the current behavior?
When resizing an event, other "preview" events are created in the state of other EventContainerWrapper, rather than just of the current event. This doesn't happen all the time, but I could not trace the pattern, as the events themselves have the same data whether it happens or not.

http://g.recordit.co/fkXXfj4uP6.gif
#### What's the expected behavior?
Only the relevant effect is resized.

Going through the source code I'm still not sure where this is coming from, so unfortunately could not submit a PR. Will be happy to get some directions as to where this problem might be.

**Comment by PeterFarnsworth23** [comment]
This happens for me as well. Sometimes when resizing a second event will appear with a null event name. only values created are random start and end times.

I noticed that this happens whenever I try to drag and drop an event and it resizes as well as moves. After fighting to bring the size of the event back down to one day for 10 plus minutes (It had became a monster event that covered 3 full months and I had to "kill" it) I deleted it. It left these random empty events. See picture. I am not sure if your events are created in a similar way or not, but that was my experience.

Thanks!

![auto created event](https://user-images.githubusercontent.com/50152001/64469759-fefb7200-d0f4-11e9-9d5f-3cc35baf2719.JPG)
